$OpenBSD: patch-iris_src_xmpp_base64_base64_cpp,v 1.1 2019/04/15 17:39:43 cwen Exp $

ppc, arm: char is unsigned by default, fix narrowing on these platforms

Index: iris/src/xmpp/base64/base64.cpp
--- iris/src/xmpp/base64/base64.cpp.orig
+++ iris/src/xmpp/base64/base64.cpp
@@ -45,7 +45,7 @@ QByteArray Base64::decode(const QString& input)
 	// 64 specifies eof
 	// everything else specifies data
 
-	char tbl[] = {
+	signed char tbl[] = {
 		-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,
 		-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,
 		-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,62,-1,-1,-1,63,
